app/api/helpers/storage.py

```diff
@@ -250,3 +252,17 @@ def generate_hash(key):
     """
     phash = generate_password_hash(key, get_settings()['secret'])
     return str(b64encode(phash), 'utf-8')[:10]  # limit len to 10, is sufficient
+
+
+def create_url(request_url, file_relative_path):
```

Comment:
Good job, this should be as close to the calling method as possible, ideally just below it